packages/eslint-plugin/src/lib/url.ts

```diff
@@ -0,0 +1,3 @@
+export function gitHubUrl(name: string) {
+  return `https://github.com/pmndrs/react-three-fiber/blob/master/packages/eslint-plugin/docs/rules/${name}.md`
```

Comment:
This is hardcoded currently. We could be more cute about it but this works good enough for now. Prone to possibly getting the name wrong. If/when this causes friction we can improve it.